frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -226,9 +226,11 @@ const Card_: React.FC<CardProps> = ({ panel, pollInterval, timespan, variables }
   const variablesJS = variables.toJS();
   const queries = rawQueries.map((expr) => evaluateTemplate(expr, variablesJS));
 
+  const isBarChart = panel.type === 'grafana-piechart-panel';
+
   return (
     <div className={`col-xs-12 col-sm-${colSpanSm} col-lg-${colSpan}`}>
-      <DashboardCard className="monitoring-dashboards__panel">
+      <DashboardCard className="monitoring-dashboards__panel" gradient={isBarChart}>
```

Comment:
Any reason not to use the gradient for any scrolling panel? I assume you only see the effect if there is content hidden.

Reply:
The gradient effect is actually added to the bottom of the panel even if there is no hidden content.

The bar chart has enough space at the bottom for the last bar to not be covered by the gradient.